### PR TITLE
Todo list add and remove

### PR DESCRIPTION
**Pull request by medsonmoombe**
## The following were implemented in this project.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
-  length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- Deleting a task update all remaining items' indexes, so they represent the current list order and are unique the first task index 
- Changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +48,19 @@ cd todo-list-first
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- [inspired by the minimalist website](https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/)
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```
Comment (ahmetbozaci) [inline README.md]: - [ ] To make a professional readme please fix the broken link.

File: src/todo-list.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    }
+    this.display();
+  }
+
+  deleteWork(workIndex) {
+    if (workIndex) {
+      this.list.splice(workIndex, 1);
+      this.display();
+    }
+  }
```
Comment (ahmetbozaci) [inline src/todo-list.js]: - [ ] After deleting todo index should update in local storage. Also, that's why if ı delete todo-2 ı cant click on todo-3 in the list. Please fix it. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should%20update%20all%20remaining%20items%27%20indexes%2C%20so%20they%20represent%20the%20current%20list%20order%20and%20are%20unique(i.e.%20if%20you%27re%20deleting%20the%20first%20task%20index%201%20from%20the%20list%2C%20the%20index%20of%20the%20next%20task(2)%20should%20set%20to%201)..)
image

File: src/index.html [hunk 3]
```diff
@@ -2,37 +2,44 @@
...
         </div>
-        <div class="todos-display">
-            
-        </div>
-        <div class="clear-todos">
-            <button class="clearAll">Clear all completed</button>
+      </section>
+      <footer>
+        <div class="complete-items clear-todos">
+          Clear all completed
```
Comment (ahmetbozaci) [inline src/index.html]: - [ ] This should clear all completed todos. It is not working. Please fix it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @medsonmoombe  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by medsonmoombe** [comment]
## The following were implemented as requested by the reviewer
- MIT link in the readme file
- the clear all completed to-do's clears all completed tasks
- After deleting todo index should update in local storage

**Review by brytebee — APPROVED** [review]
Hi, @medsonmoombe👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**